webapp-django/crashstats/settings/base.py

```diff
@@ -447,6 +447,7 @@ def path(*dirs):
         'LESS_BINARY',
         path('node_modules/.bin/lessc')
     ),
+    'LESS_ARGUMENTS': '--global-var="base-path=\'/tmp/crashstats-static/crashstats/css/base\'"',
```

Comment:
I get the following error when running `make dockerbuild` to build a new base image:

```
pipeline.exceptions.CompilerError: ['/webapp-frontend-deps/node_modules/.bin/lessc', "--global-var=base-path='/tmp/crashstats-static/crashstats/css/base'", u'/app/webapp-django/static/home/css/home.less'] exit code 1
FileError: '/tmp/crashstats-static/crashstats/css/base/variables.less' wasn't found. Tried - /tmp/crashstats-static/crashstats/css/base/variables.less,/app/webapp-django/static/home/css/tmp/crashstats-static/crashstats/css/base/variables.less in /app/webapp-django/static/home/css/home.less on line 2, column 1:
1 @base-path: '/tmp/crashstats-static/crashstats/css/base';
2 @import "@{base-path}/variables.less";
```

I suspect it has to do with the path being passed here. I believe `/tmp/crashstats-static` only exists during local development and not during the build process. The [note on the webapp docs](http://socorro.readthedocs.io/en/latest/components/webapp.html) mentions that normally these files are in `/app/webapp-django/static`, maybe that would work instead?

Reply:
Thanks for pointing this out, I knew there was something wrong with the `/tmp` business.

I've changed this so it checks if the site is running in dev mode, and switches between the two directories appropriately. This seems to satisfy the build and it works locally, hopefully this approach is acceptable!